project/bin/.settings/org.eclipse.core.resources.prefs

```diff
@@ -0,0 +1,3 @@
+eclipse.preferences.version=1
```

Comment:
all these eclipse files are your IDE settings? if so we should gitignore these

Reply:
how do i gitignore ?

Reply:
you  probably want to ignore any bin/ folders as those are created by ide settings like these: https://www.atlassian.com/git/tutorials/saving-changes/gitignore 
try adding **/bin to the .gitignore file? 

(also this contains a section on ignoring previously commited files so check that one out to remove these from the PR)